### PR TITLE
Add the 'null' value type to the CheckBox.value description

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ default: false
 Specifies the UI component state.
 
 ---
-The check box can be in one of the following states depending on the **value** property value.
+The check box can be in one of the following states depending on the **value** property's value.
 
 - checked  
  value is **true**
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md]:
```suggestion
 **value** is **true**
```

File: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ The check box can be in one of the following states depending on the **value** p
  value is **false**
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md]:
```suggestion
 **value** is **false**
```

File: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ The check box can be in one of the following states depending on the **value** p
...
 
 - undetermined  
- value is **undefined**
+ value is **undefined** or **null**
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md]:
```suggestion
 **value** is **undefined** or **null**
```

File: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md [hunk 4]
```diff
@@ -17,9 +17,9 @@ The check box can be in one of the following states depending on the **value** p
...
 
-An end-user can change the UI component value using the **Space** key.
+An end-user can use the **Space** key to change the UI component value.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md]:
```suggestion
Users can press the Space key to change the **value**.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md [hunk 5]
```diff
@@ -8,18 +8,18 @@ default: false
...
 - unchecked  
- value is **false**
+ **value** is **false**
 
 - undetermined  
- value is **undefined**
+ **value** is **undefined** or **null**
 
-An end-user can change the UI component value using the **Space** key.
+Users can press the Space key to change the **value**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md]: Can we add bold font to "Space" as well? For example:

`Users can press the **Space** key to change the **value**.`

**Pull request merged**